components/brave_welcome_ui/assets/brave-search-cube.svg

```diff
@@ -0,0 +1,79 @@
+<svg width="218" height="182" viewBox="0 0 218 182" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
did you optimize this svg? https://jakearchibald.github.io/svgomg/

Reply:
Thanks! done.